src/pages/index.tsx

```diff
@@ -86,6 +95,9 @@ const AdminPage: NextComponentType = () => {
                     {configItem.key}
                   </label>
                   <input type="text" className="form-control" id={configItem.key} placeholder="Team name" value={configItem.value} disabled />
+                  <button type="button" className="btn btn-primary mb-2 mt-3" onClick={handleChange}>
+                    Change
+                  </button>
```

Comment:
This is where you'll need to pass the key and new value to your `handleChange` function. I would also change the button text to reflect exactly what it's doing.

```suggestion
                  <button type="button" className="btn btn-primary mb-2 mt-3" onClick={() => handleChange(configItem)}>
                    {configItem.value === 'true' ? 'Disable' : 'Enable'}
                  </button>
```